Source/stores.cpp

```diff
@@ -738,7 +738,7 @@ void S_StartSRepair()
 	stextsize = 1;
 	repairok = FALSE;
 	storenumh = 0;
-	for (i = 0; i < 40; i++)
+	for (i = 0; i < 48; i++)

```

Comment:
Afaik 48 isn't an enum

Reply:
yep but fixed it while i was on it :P

Reply:
Was this causing diff in vanilla?

Reply:
@AJenbo it only changed a global address, that's why nobody noticed

Reply:
:( hope we don't have much of this